wagtailio/static/sass/components/_blog.scss

```diff
@@ -38,7 +38,13 @@
 
         @include media-query(medium) {
             padding-bottom: 80px;
+            padding-top: 80px;
         }
+
+        @include media-query(large) {
+          padding-bottom: (20vh);
```

Comment:
I'm being very picky, but the indentation looks slightly too small here?

Reply:
have a super weird issue on this build, editorconfig specifies 2 spaces for indentation on scss, but all of the code is spaced to 4… I suspect I'm going to change the editorconfig to 4 to prevent me going insane